cmd2/argparse_completer.py

```diff
@@ -422,9 +434,35 @@ def consume_positional_argument() -> None:
                 completion_results = self._complete_for_arg(pos_action, text, line, begidx, endidx, consumed)
                 if not completion_results:
                     self._print_action_help(pos_action)
+                elif len(completion_results) > 1:
+                    completion_results = self._format_completions(pos_action, completion_results)
 
         return completion_results
 
+    def _format_completions(self, action, completions: List[Union[str, CompletionItem]]):
```

Comment:
Recommend adding a unit test or two for the new stuff added to argparse_completer.py